rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
Please add Getters and Setters for `@book` and `@person` instance variables in `Rental` class.